tests/detect.bats

```diff
@@ -21,6 +26,21 @@ load test-helper
   assert_success
 }
 
+@test "$clinom detect rpi full" {
+  run "${clicmd}" detect rpi model
+  assert_success
+}
+
+@test "$clinom detect rpi (invalid second argument)" {
+  run "${clicmd}" detect rpi foobar
+  assert_failure && assert_output --partial "commands supported"
+}
+
+@test "$clinom detect <option> <an extra unnecessary option>(too many arguments)" {
```

Comment:
Could add a space between `<an extra unnecessary option>` and `(too many arguments)` so that it reads:

> <an extra unnecessary option> (too many arguments)